app/code/BVMyBlog/Blog/Model/ResourceModel/Post.php

```diff
@@ -0,0 +1,20 @@
+<?php
```

Comment:
Please refer to app/code/BVMyBlog/Blog/Model/ResourceModel/Blog.php. Why do we need this model. They point to the same table/ Please remove one of them